app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java

```diff
@@ -1,5 +1,7 @@
 package com.sdpteam.connectout.event;
 
+import android.util.Log;
```

Comment:
Maybe this import is useless

Reply:
Indeed! Forgot to remove it after my debugging